backends/firrtl/test.sh

```diff
@@ -1,22 +1,25 @@
 #!/bin/bash
 set -ex
 
-../../yosys -p 'prep -nordff; write_firrtl test.fir' test.v
+cd ../../
+make
+cd backends/firrtl
```

Comment:
Maybe `make -C ../..` would be better? But tbh I personally am not a big fan of this either way. I'd prefer just using something like `make -C ../.. -j$(nproc) && bash test.sh` when calling it, iff I want to re-run make before running the test. But you have to use this script far more frequently than I, so do whatever you like :), I just could not resist to comment..